src/components/common/PractitionerActionForm.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+ 
+import closeIcon from '../../assets/images/close-icon.png'
+
+const PractitionerActionForm = (props: any) => {
+
+    const handleMenuClose = (e:any) => {
+        e.stopPropagation();
+        e.preventDefault();
+        props.setIsVisible(false);
+    }
+
+  return (
+       <div className="practitionerActionForm__modal modal">
+            <div className="practitionerActionForm__header mb-md">
+                <span className="text__title-med">Add Practitioner</span>
+                <img src={closeIcon} alt="" onClick={handleMenuClose}/>
+            </div>
+            <div className="practitionerActionForm__content">
+            <input type="text" className="input__text mb-tn" placeholder='YOUR MAIL'/>
+            <input type="text" className="input__text mb-tn" placeholder='YOUR MAIL'/>
+            <input type="text" className="input__text mb-tn" placeholder='YOUR MAIL'/>
+            <input type="text" className="input__text mb-tn" placeholder='YOUR MAIL'/>
```

Comment:
is it inprogress? why duplicates? Can we simply write 'Email id' instead? so that user won't be confused with their mailbox? and can we add some padding y axis?